listeners/test-client.js

```diff
@@ -0,0 +1,114 @@
+'use strict';
```

Comment:
Not related to this line, but add 'listeners/**/*.js' to jshint in the Gruntfile (line 24)